backend/test/api/player_view_functions_test.clj

```diff
@@ -9,13 +10,27 @@
   (expect
     [{:location [:hand] :owner "opp"}
      {:power -1 :location [:row 0] :owner "opp"}
-     {:power 1 :attr "kill" :location [:hand] :owner "me"}
+     {:power 1 :location [:hand] :owner "me"}
      {:power 100 :location [:row 1] :owner "me"}]
     (functions/get-cards {:cards [{:power 99 :location [:hand] :owner "opp_name"}
                                   {:power -1 :location [:row 0] :owner "opp_name" :secret-attr "who knows?"}
-                                  {:power 1 :attr "kill" :location [:hand] :owner "me_name"}
+                                  {:power 1 :internal-attr "kill" :location [:hand] :owner "me_name"}
                                   {:power 100 :location [:row 1] :owner "me_name"}]}
-                        "me_name")))
+                        "me_name"))
+
+  ; Correctly returns cards with abilities
+  (expect
+    [{:power 1 :location [:hand] :owner "me" :description "Enhance 100" :target 1}
+     {:power 17 :location [:hand] :owner "me" :description "Enhance -1" :target 1}
```

Comment:
Again, I feel like "Enhance -1" just looks very weird. Diminish or Weaken maybe? It might even then be positive "Weaken 1"